templates/webpack-default/app/controllers/index.js

```diff
@@ -0,0 +1,5 @@
+const tabgroup = require('@/tabgroup');
+
+tabgroup.tg = $.index;
```

Comment:
Could change this without assigning a variables and setting it straight to the source 
```js
require('@/tabgroup').tg = $.index
```

Reply:
I've updated this file as it's a single usage, but kept the one in profile/index.js as it's used in the click handler